<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogtoolbox</class>
 <widget class="QDialog" name="dialogtoolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>208</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>客户信息管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <attribute name="title">
         <string>通讯录管理</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="1" colspan="3">
            <widget class="QLineEdit" name="edtTags"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>省份：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="edtRemark"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>标签：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>城市：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="edtSex"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>备注：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="edtName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>性别：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="edtCity"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>类型：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edtProv"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="edtType"/>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>名字：</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="4">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnSearch">
               <property name="text">
                <string>根据条件过滤</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnInvalid">
               <property name="text">
                <string>所有失效</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNoremark">
               <property name="text">
                <string>所有易混淆</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNotags">
               <property name="text">
                <string>所有无标签</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtProv</tabstop>
  <tabstop>edtCity</tabstop>
  <tabstop>edtSex</tabstop>
  <tabstop>edtRemark</tabstop>
  <tabstop>edtType</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtTags</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
